.gitignore

```diff
@@ -6,4 +6,15 @@ platforms/
 plugins/
 ionic.project
 .io-config.json
+
+#Mac users
 .DS_Store
+*/.DS_Store
```

Comment:
I don't think you need to add this line. The first line should take care of it in any directory. I think the issue just was that the .DS_Store file was never deleted.